src/api/server.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export async function fetchTodoList() {
+    const response = await fetch('http://127.0.0.1:8000/todo-list/')
+    const actualData = response.json()
+    return actualData
+}
+
+export async function addTodo(todo: any) {
+    const response = await fetch('http://127.0.0.1:8000/todo', {
```

Comment:
You set the response but ignore it. I'd expect response to be returned.

Reply:
I'm going to return the boolean from `response.ok` to help determine whether or not to add in a new todo.